lib/matplotlib/patches.py

```diff
@@ -942,18 +943,18 @@ def __init__(self, values, edges, *,
             The direction of the steps. Vertical means that *values* are along
             the y-axis, and edges are along the x-axis.
 
-        baseline : float or None, default: 0
-            Determines starting value of the bounding edges or when
+        baseline : float, 1D array-like or None, default: 0
+            Determines bottom value of the bounding edges or when
```

Comment:
Minor style issue:
```suggestion
            The bottom value of the bounding edges or when
```

What happens if fill is False, but you have an array baseline? I guess you get a closed polygon. That difference should be mentioned vs. the scalar baseline for fill=False.

Same in `stairs()`.